### ISSUE TITLE
Allow napari to launch in foreground (when called from Jupyter lab)

### ISSUE THREAD

**Issue by s-m-e** [napari/napari#735]
## 🚀 Feature

A simple option when calling `napari.view_image`: `foreground`, a bool. If `True`, the `napari` viewer window receives the focus automatically when being opened.

## Motivation

At least when launched from Jupyter, `napari` opens its windows in the background. I can easily switch to them using my desktop's task manager / task bar, but this quickly becomes annoying with many open tasks / windows, virtual desktops and multiple physical screens. When I call `napari`, I want to directly see it in foreground, interact with it and close it, getting back to where I came from.

## Pitch

Less looking for windows, quicker workflow.

## Alternatives

None.

## Additional context

None.

**Comment by tlambert03** [comment]
hi @s-m-e 
that behavior was actually changed in #721 and merged into master in 84e05db (not yet released).

however, it came at the cost of a reversion of #553... causing the menubar to be inactive when launching _outside_ of a jupyter notebook.  See ongoing discussion in #732.  We're not immediately sure how to reconcile the two problems... but regardless of which way the main branch goes with it, if you want to fix it manually in your notebook, you can do it like this:
```python

with napari.gui_qt():
    viewer = napari.view_image(data)
    viewer.window._qt_window.raise_()
```

**Comment by sofroniewn** [comment]
As noted above this was fixed in #721 and preserved by #795 which fixed some related menubar problems. I am closing now
